### PR TITLE
Release 1 - basic routes and migrations/seeds

### PR DESCRIPTION
**Pull request by RonnySAlvarado**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status
- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: backend/api/models/ReviewModel.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+module.exports = {
+  getAll_reviews,
+  add
+};
+
+function getAll_reviews() {
+  return db("reviews");
+}
+
+function add(review) {
```
Comment (MrT3313) [inline backend/api/models/ReviewModel.js]: might make senses to keep our function naming consistent across models. 

type_table --> getAll_reviews // get_review // add_review // update_user etc etc // delete_user

File: backend/api/routes/usersRoute.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+router.get("/", async (req, res) => {
+  try {
+    const allUsers = await USERS_MODEL.getAll_users();
+    res.status(200).json(allUsers);
+  } catch (err) {
+    res.status(500).json({ msg: err });
+  }
+});
+
+// @route  GET users/:userid
```
Comment (MrT3313) [inline backend/api/routes/usersRoute.js]: random....is there a reason you are using @before each comment?

File: backend/seeds/003_reviews.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        {
+          rating: 2,
+          comments: "Mediocre chicken.",
+          user_id: 1,
+          location_id: 1
+        },
+        {
+          rating: 3,
+          comments: "Best chicken I've ever had!",
+          user_id: 1,
```
Comment (Memitaru) [inline backend/seeds/003_reviews.js]: Obviously not a huge deal for release one but we generally won't want two reviews of the same location by the same user (I assume). Something we'll want to consider when we get to the add review stage.

Comment (RonnySAlvarado) [reply backend/seeds/003_reviews.js]: I thought about this, and I don't think there's anything wrong with the person not liking the stuff the first time around and then coming back later on and potentially liking it then. That would be the only case I can think of where 2 reviews by the same user of the same location would be used.

Ultimately, you're right though. We can figure that out later on if we want to put that limitation there.

File: backend/api/routes/usersRoute.js (hunk 2 above)
Comment (RonnySAlvarado) [reply backend/api/routes/usersRoute.js]: It's just a convention I'm trying out that some devs I'm watching do (TraversyMedia). We can always revert to something else.

File: backend/api/models/ReviewModel.js (hunk 1 above)
Comment (RonnySAlvarado) [reply backend/api/models/ReviewModel.js]: Agreed, this is something I've been meaning to talk with the group about to see how we like to keep these naming conventions

File: backend/seeds/003_reviews.js (hunk 3 above)
Comment (davidaceves) [reply backend/seeds/003_reviews.js]: What about just updating the original review? Or at least some kind of limit on how many reviews one user can submit.

**Pull request merged**